internal/criteria/criteria.go

```diff
@@ -0,0 +1,11 @@
+package criteria
+
+// CollabStyle ..
```

Comment:
Why the comment?

Reply:
This is more out of annoyance. The linter in VScode keeps complaining when you don't put a comment. I don't know how to turn off the linter rule. I'm ok to delete them. I mostly use goland, so it's not much of an issue for me.

Reply:
There got to be a way. I would say it is ok to keep them in this PR, 
and the next PR add `.vscode/settings.json` to turn it off and remove these comments.
And in the travis turn on any linter check. I know that go compiler do some linting itself, but don't know about which linter the vscode extension based on.

Reply:
I have looked about it in the past, but wasn't able to find a way. I ended up just creating a snippet that added them automatically.

Reply:
Are the comments supposed to be doc generating comments?

Reply:
Are these comments supposed to be doc generating comments?

Reply:
Ok, I see these are required. We should probably add godoc to the travis yaml.
https://golang.org/doc/effective_go.html#commentary

Since these comments are actually treated like code, they probably don't break the clean code "no comment" rules. And we should probably fill them in.  Thoughts?